Rover/mode_guided.cpp

```diff
@@ -3,11 +3,14 @@
 
 bool ModeGuided::_enter()
 {
-    // set desired location to reasonable stopping point
-    if (!g2.wp_nav.set_desired_location_to_stopping_location()) {
```

Comment:
you should still call this to set the stopping point for companion computer and other monitoring system, otherwise they won't know where is the target position.

Reply:
@khancyr,

Thanks for the feedback.

The problem though with calling this is that we are not using it for control anymore (if this PR goes in).  The vehicle will slow down at the configured deceleration rate but laterally it does not have a specific location.